aws-opentelemetry-distro/src/amazon/opentelemetry/distro/_aws_attribute_keys.py

```diff
@@ -16,3 +16,8 @@
 AWS_QUEUE_URL: str = "aws.sqs.queue_url"
 AWS_QUEUE_NAME: str = "aws.sqs.queue_name"
 AWS_STREAM_NAME: str = "aws.kinesis.stream_name"
+AWS_BEDROCK_DATASOURCE_ID: str = "aws.bedrock.datasource_id"
+AWS_BEDROCK_KNOWLEDGEBASE_ID: str = "aws.bedrock.knowledgebase_id"
+AWS_BEDROCK_AGENT_ID: str = "aws.bedrock.agent_id"
+AWS_BEDROCK_GUARDRAIL_ID: str = "aws.bedrock.guardrail_id"
+AWS_BEDROCK_RUNTIME_MODEL_ID: str = "gen_ai.request.model"
```

Comment:
q: can we define all `gen_ai.*` attributes here with the same comments we left in java sdk impl?

Reply:
I don't think we should do this, instead import from upstream. Resolving.